core/trino-spi/src/main/java/io/trino/spi/ptf/TableFunctionProcessor.java

```diff
@@ -15,33 +15,64 @@
 
 import io.trino.spi.Experimental;
 import io.trino.spi.Page;
+import io.trino.spi.connector.ConnectorSplit;
 
 import java.util.List;
 import java.util.Optional;
 
 @Experimental(eta = "2023-03-31")
 public interface TableFunctionProcessor
```

Comment:
What's the purpose of this interface? It has no methods.

Reply:
removed